xbmc/Util.cpp

```diff
@@ -2172,20 +2167,20 @@ bool CUtil::FindVobSubPair(const std::vector<std::string>& vecSubtitles, const s
     std::string strIdxFile;
     std::string strIdxDirectory;
     URIUtils::Split(strIdxPath, strIdxDirectory, strIdxFile);
-    for (unsigned int j=0; j < vecSubtitles.size(); j++)
+    for (const auto &j : vecSubtitles)
```

Comment:
I'd appreciate it if the variable names were a bit more verbose (also below). Single-char is OK for loop indices because when you refer to the element you have the context of the array anyway. But for actual elements here I think it is not enough.